CHANGELOG.md

```diff
@@ -41,6 +41,7 @@
 
 ### Improvements
 
+- **General:** Using manager client in KEDA Metrics Server to avoid flush request to Kubernetes Apiserver([2914](https://github.com/kedacore/keda/issues/2914))
```

Comment:
Sorry for being a pain... Could you sort this line inside `**General:**` entries?